src/sentry/integrations/gitlab/integration.py

```diff
@@ -184,6 +200,62 @@ def search_issues(self, query: str | None, **kwargs) -> list[dict[str, Any]]:
         assert isinstance(resp, list)
         return resp
 
+    def get_pr_comment_workflow(self) -> PRCommentWorkflow:
+        return GitlabPRCommentWorkflow(integration=self)
+
+
+MERGED_PR_COMMENT_BODY_TEMPLATE = """\
+## Suspect Issues
+This pull request was deployed and Sentry observed the following issues:
+
+{issue_list}"""
+
+MERGED_PR_SINGLE_ISSUE_TEMPLATE = "- ‼️ **{title}** `{subtitle}` [View Issue]({url})"
+
+
+class GitlabPRCommentWorkflow(PRCommentWorkflow):
```

Comment:
i'm curious if most of these functions will be different when it comes to other integrations, we can pull the duplicate ones to the parent eventually